src/Lecture4.hs

```diff
@@ -222,8 +269,12 @@ Have a look at the 'sconcat' function from the 'Semigroup' typeclass to
 implement the next task.
 -}
 
+
+strictSconcat:: (Semigroup a) => NonEmpty a -> a
+strictSconcat = foldr1 (\x !acc -> x <> acc)
+
 combineRows :: NonEmpty Row -> Stats
-combineRows = error "TODO"
+combineRows rows = strictSconcat $ fmap rowToStats rows
```

Comment:
Great usage of `Functor` instance for `NonEmpty` to map values 💎 
You can use function composition and eta-reduce the implementation if you like:

```suggestion
combineRows = strictSconcat . fmap rowToStats rows
```